liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -333,3 +333,11 @@ warn.install.license.useraction=Download and reinstall the Liberty license jar f
 error.install.license=CWWKM2183E: Failed to install the Liberty production license {0}. Return code is {1}.
 error.install.license.explanation=Failed to install the Liberty production license to the runtime installation. The current installation might contain features that are not supported by the production license terms and conditions.
 error.install.license.useraction=Check the return code and execution failure log to investigate the cause of the failure. If the current installation contains features which are not supported by the Liberty production license terms and conditions, follows the instruction in the execution failure log to uninstall these features and try again. You can also install a different production license that supports these features.
+
+warn.invalid.app.directory=CWWKM2184W: Invalid value "{0}" for the liberty-maven-plugin configuration parameter "appsDirectory" is detected. Default value will be used.
```

Comment:
CWWKM2184W: The project POM liberty-maven-plugin configuration has an invalid value "{0}" for the "appsDirectory" parameter. The default appsDirectory value will be used.